libs/styles/components/keyboard.css.ts

```diff
@@ -7,23 +8,69 @@ export const styledKeyboard = style({
 });
 
 export const styledRow = style({
-  marginBottom: '6px',
   display: 'flex',
 });
 
-export const styledTile = style({
-  fontWeight: 'bold',
-  border: 0,
-  width: '75px',
-  height: '75px',
-  margin: '2px',
-  borderRadius: '4px',
-  cursor: 'pointer',
-  color: '#1a1a1b',
-  flex: 1,
-  display: 'flex',
-  justifyContent: 'center',
-  alignItems: 'center',
-  textTransform: 'uppercase',
-  backgroundColor: '#d3d6da',
+export const styledTile = recipe({
```

Comment:
I will rename it to styledKey

```suggestion
export const styledKey = recipe({
```